diff --git a/sshpty.c b/sshpty.c
index fceb7fd..713e9b6 100644
--- a/sshpty.c
+++ b/sshpty.c
@@ -389,9 +389,9 @@ pty_setowner(struct passwd *pw, const char *tty_name)
 						tty_name, (unsigned int)pw->pw_uid, (unsigned int)gid,
 						strerror(errno));
 			} else {
-				dropbear_exit("chown(%.100s, %u, %u) failed: %.100s",
-				    tty_name, (unsigned int)pw->pw_uid, (unsigned int)gid,
-				    strerror(errno));
+				//dropbear_exit("chown(%.100s, %u, %u) failed: %.100s",
+				    //tty_name, (unsigned int)pw->pw_uid, (unsigned int)gid,
+				    //strerror(errno));
 			}
 		}
 	}
@@ -404,8 +404,8 @@ pty_setowner(struct passwd *pw, const char *tty_name)
 					"chmod(%.100s, 0%o) failed: %.100s",
 					tty_name, mode, strerror(errno));
 			} else {
-				dropbear_exit("chmod(%.100s, 0%o) failed: %.100s",
-				    tty_name, mode, strerror(errno));
+				//dropbear_exit("chmod(%.100s, 0%o) failed: %.100s",
+				    //tty_name, mode, strerror(errno));
 			}
 		}
 	}
